views/reset-password/reset.pug

```diff
@@ -0,0 +1,24 @@
+html
```

Comment:
The frontend is completely out of the scope of this task. Why would you do this, even though you know we are still working on the backend ?

Reply:
fixed